packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.tsx

```diff
@@ -0,0 +1,542 @@
+                reviewerIds!.map(id => {
+                  return <UserOrTeamBadge key={id} principalId={id} />
+                })}
+              {!isLoadingACL && isEmpty(reviewerIds) && (
+                <UserOrTeamBadge principalId={ACT_TEAM_ID} />
+              )}
+            </Typography>
+            <br />
+          </>
+        )}
+        <Typography variant="dataFieldKey" gutterBottom>
+          Conditions
+        </Typography>
+        {accessRequirement ? (
+          <Typography variant="smallText1" component={'div'} gutterBottom>
+            <ul>
+              <li>
+                Expiration period:{' '}
+                {dayjs
+                  .duration({
+                    milliseconds: accessRequirement.expirationPeriod,
+                  })
+                  .asDays()}{' '}
+                day(s)
+                {accessRequirement.expirationPeriod === 0 && ' (no expiration)'}
+              </li>
+
+              {accessRequirement.isCertifiedUserRequired && (
+                <li>User must be Certified</li>
+              )}
+              {accessRequirement.isValidatedProfileRequired && (
+                <li>User Profile must be Validated</li>
+              )}
+              {accessRequirement.isDUCRequired && <li>DUC is required</li>}
+              {accessRequirement.isIDURequired && <li>IDU is required</li>}
+              {accessRequirement.isIDUPublic && (
+                <li>IDU will be made public</li>
+              )}
+              {accessRequirement.isIRBApprovalRequired && (
+                <li>IRB Approval is required</li>
+              )}
+              {accessRequirement.areOtherAttachmentsRequired && (
+                <li>Other attachments are required</li>
+              )}
+            </ul>
+          </Typography>
+        ) : (
+          <Skeleton width={100} />
+        )}
+        <br />
+        <div className="SubmissionSummaryGrid">
+          <Typography variant="dataFieldKey">Submitted By</Typography>
+          <Typography variant="smallText1">
+            {submission ? (
+              <UserBadge userId={submission.submittedBy} />
+            ) : (
+              <Skeleton width={100} />
+            )}
+          </Typography>
+          <Typography variant="dataFieldKey">Submitted On</Typography>
+          <Typography variant="smallText1">
+            {submission ? (
+              formatDate(dayjs(submission.submittedOn))
+            ) : (
+              <Skeleton width={100} />
+            )}
+          </Typography>
+          {isReviewer && (
+            <>
+              <Typography variant="dataFieldKey">Modified By</Typography>
+              <Typography variant="smallText1">
+                {submission ? (
+                  <UserBadge userId={submission.modifiedBy} />
+                ) : (
+                  <Skeleton width={100} />
+                )}
+              </Typography>
+            </>
+          )}
```

Comment:
Hide last modified field (could be a reviewer) unless the current user is a reviewer